my-code/whats-in/src/pages/Homepage/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import iconSearch from '../../images/icon-magnifier-grey.svg'
+import iconLike from '../../images/icon-heart-grey.svg'
+import iconLiked from '../../images/icon-heart-full.svg'
+import Loading from '../../components/Loading'
+import Empty from '../../components/Empty'
+import NotFound from '../../components/Notfound'
+
+export default function Homepage({ handleSearch, movies, handleMovieSelection, isLoading, handleLike, isMovieLiked, isEmptySearch }) {
+
+    const handleChange = (event) => {
```

Comment:
the search network call should be debounced somehow (there are several valid approaches).

To understand why the current approach might be problematic:
1. Open up your browser's inspector and focus on the network tab
2. Focus your input and write something like "The godfather"
3. Check how many requests were fired to search for movies (one for each character input)

In this cases we want to wait for the user to stop typing _and then_ fire the search network call.